plugins/modules/mso_user.py

```diff
@@ -256,7 +259,10 @@ def main():
                     mso.existing = mso.proposed
                 else:
                     mso.existing = mso.request(path, method='PUT', data=mso.sent)
+
         else:
+            if user_password is None:
+                mso.fail_json("No password found. Please enter a password to create the user.")
```

Comment:
"The user {0} does not exist. The 'user_password' attribute is required to create a new user.".format(user_name)